### PR TITLE
upgrade API client connector for PF

### PR DESCRIPTION
**Pull request by kahboom**
Epic: [[ENTESB-12896] Migration from PF3 to PF4 for Fuse Online](https://issues.redhat.com/browse/ENTESB-12896)

### Before

Screenshot 2020-04-14 17 35 42

### After

Screenshot 2020-04-14 13 25 25

Screenshot 2020-04-14 17 48 14

Screenshot 2020-04-14 17 48 34

Screenshot 2020-04-14 17 48 52

cc @gashcrumb

### CODE REVIEW

**Comment by gashcrumb** [comment]
Nice!

Yeah, I think for the API security form it could be right-justified.  For the form I set it explicitly [like this](https://github.com/syndesisio/syndesis/pull/8169/files#diff-c27ce893f3be3b62c51ba3b6f8522692R73-R75), I didn't really see a better way.

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiClientConnectorCreateSecurity.tsx [hunk 1]
```diff
@@ -105,11 +103,11 @@ export const ApiClientConnectorCreateSecurity: React.FunctionComponent<IApiClien
     setValid(isValid(selectedType, authUrl, newUrl));
   };
   return (
-    <Card style={{ maxWidth: '600px', margin: ' auto' }}>
-      <CardHeader>
+    <Stack style={{ maxWidth: '600px', margin: ' auto' }} gutter="md">
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiClientConnectorCreateSecurity.tsx:106]: Yeah, maybe lose the `margin` here and let it right-justify, that seems to be the norm in the pf4 wizard.

File: app/ui-react/syndesis/src/modules/connections/pages/create/ConfigurationPage.tsx [hunk 2]
```diff
@@ -207,20 +206,6 @@ export const ConfigurationPage: React.FunctionComponent = () => {
                     i18nNameConnection={t('connections:create:review:title')}
                   />
                 }
-                footer={
```
Comment (gashcrumb) [inline app/ui-react/syndesis/src/modules/connections/pages/create/ConfigurationPage.tsx:210]: Was this supposed to come out?

**Comment by gashcrumb** [comment]
I think also you may want to see if you can create a component for the wizard footer, that'll make it more consistent with the create connection wizard and also keep the buttons in a consistent location.  I suspect on the apicurio page you wouldn't use it and probably still have to put a button on the breadcrumb area.

**Comment by kahboom** [comment]
After changes:

Screenshot 2020-04-15 09 45 35

**Comment by gashcrumb** [comment]
Ah you'll have to rebase this, then I can approve again.

**Comment by gashcrumb** [comment]
Lint error:

```
[INFO] lerna ERR! yarn run build stderr:
[INFO] (rpt2 plugin) Error: /workspace/app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiClientConnectorCreateSecurity.tsx(13,1): semantic error TS6133 'ButtonLink' is declared but its value is never read.
[INFO] Error: /workspace/app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiClientConnectorCreateSecurity.tsx(13,1): semantic error TS6133 'ButtonLink' is declared but its value is never read.
[INFO]     at error (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup/dist/rollup.js:3460:30)
[INFO]     at Object.error (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup/dist/rollup.js:21251:17)
[INFO]     at RollupContext.error (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:17186:30)
[INFO]     at lodash_2 (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24943:23)
[INFO]     at arrayEach (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:532:11)
[INFO]     at forEach (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:9360:14)
[INFO]     at printDiagnostics (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24916:5)
[INFO]     at Object.transform (/workspace/app/ui-react/node_modules/microbundle/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:26582:17)
[INFO]     at /workspace/app/ui-react/node_modules/microbundle/node_modules/rollup/dist/rollup.js:20962:25
```

**Comment by kahboom** [comment]
retest this please 👯

**Comment by kahboom** [comment]
retest this please

**Comment by kahboom** [comment]
retest this please

**Comment by mkralik3** [comment]
@kahboom  Can this PR be backported to the 1.10.x? It looks that the backport bot doesn't work.

**Comment by kahboom** [comment]
Yes, thanks for bringing my attention to this @mkralik3, I'll backport this manually

**Pull request merged**